compiler.js

```diff
@@ -180,6 +180,34 @@ class ForLoop extends Node {
   }
 }
 
+class If extends Node {
+  constructor({condition, statements}) {
+    super();
+    this.condition = condition;
+    console.log (this.condition);
+    this.statements = statements;
+  }
+  toAssembly(symbolTable) {
+    const childInstructions = this.statements.map(s => s.toAssembly(symbolTable)).flat();
+    const jumpOperations = {
```

Comment:
We should (eventually) extract this out since this duplicates the `jumpOperations` defined in `ForLoop`.  Fine for now though!